agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1560,6 +1561,9 @@ bool main_thread::handle_1905_1_message(ieee1905_1::CmduMessageRx &cmdu_rx,
         return handle_1905_discovery_query(cmdu_rx);
         //LOG(INFO) << "I got the Topology Query message!";
     }
+    case ieee1905_1::eMessageType::AP_CAPABILITY_QUERY_MESSAGE: {
```

Comment:
I mention it on #161, this handler should not be here, it should be in son_slave.
It doesn't matter that it doesn't have TLVs. Channel preference query is also a message without TLVs and being handled in the son_slave which is responsible for the radio.
The backhaul manager automatically forwards messages to the son_slaves if not handled in the backhaul manager itself.
I don't see the point of handling it here and then create new CMDU to send it to the son_slave.